modules/augment/src/rotate.cpp

```diff
@@ -0,0 +1,40 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#include "precomp.hpp"
+namespace cv {
+namespace augment {
+
+    Rotate::Rotate(Range _angleRange)
+    {
+        this->angleRange = _angleRange;
+        this->resetRandom();
+    }
+
+    Rotate::Rotate() : Rotate::Rotate(Range(0, 180)) {}
+
+    void Rotate::resetRandom()
+    {
+        this->currentAngle = rng.uniform(this->angleRange.start, this->angleRange.end + 1);
```

Comment:
Why `+1`? If user specified the range one expect the values only from this range. Angle should be continuous value.

Reply:
resolved that ( will push the new code as soon as I'm finished with all the issues)